### PR TITLE
Updated PD JS3 W2

### PR DESCRIPTION
**Pull request by esma-g**
## What does this change?

Module: JS3
Week(s): W2

## Checklist

- [x] I have read the [contributing guidelines](https://syllabus.codeyourfuture.io/contributing/overview)
- [x] I have checked my spelling and grammar with an [automated tool](https://www.grammarly.com/grammar-check)
- [x] I have previewed my changes to check the [markdown renders](https://docs.github.com/en/get-started/writing-on-github/getting-started-with-writing-and-formatting-on-github/basic-writing-and-formatting-syntax) as I intend
- [x] I have run my code to check it works
- [x] My changes follow our [Style Guide](https://syllabus.codeyourfuture.io/guides/code-style-guide)

## Description

Updated PD JS3 W2

## Who needs to know about this?

@kfklein15 @CodeYourFuture/syllabus-team

### CODE REVIEW

File: docs/js-core-3/week-2/lesson.md [hunk 1]
```diff
@@ -399,40 +399,92 @@ fetch(_____)
...
 
 :::
 
-[Agile: Planning and preparation](https://docs.google.com/presentation/d/1TJNjPOha2P5cWW6IO9jAc9GTvsosqVGkkG5pAbbvfg4/edit#slide=id.g1b740a9f8c6_0_336)
+**Definitions**
+
+**Positive feedback**: This is the type of feedback that we all want to hear, it’s when someone praises our work.
+
+Appreciation and positive remarks in the workplace can help an employee feel appreciated, can help to reinforce good behaviours and will help employees with their professional development.
+However, it is important not to overuse positive feedback as its value will decrease. You may find it more valuable to hear positive comments from someone who almost never praises you for your work than a coworker constantly telling you how awesome you are. Too much positive feedback can also make employees complacent and feel less challenged in their roles.
```
Comment (illicitonion) [inline docs/js-core-3/week-2/lesson.md]:
```suggestion
Appreciation and positive remarks in the workplace can help an employee feel appreciated, can help to reinforce good behaviours and will help employees with their professional development.

However, it is important not to overuse positive feedback as its value will decrease. You may find it more valuable to hear positive comments from someone who almost never praises you for your work than a coworker constantly telling you how awesome you are. Too much positive feedback can also make employees complacent and feel less challenged in their roles.
```

In markdown, you need a whole empty line between paragraphs :)

Comment (illicitonion) [inline docs/js-core-3/week-2/lesson.md:424]: I realise this isn't an exhaustive guide, but I'd think about including something about how _specific_ positive feedback can be much more effective than general positive feedback.

"They way you presented graphs to show our progress really made it easy to notice what tasks I needed to focus on which helped get the project delivered" vs "It was great working with you, it helped the project succeed"

**Review by illicitonion — DISMISSED** [review]

File: docs/js-core-3/week-2/lesson.md (hunk 1 above)
Comment (kfklein15) [reply docs/js-core-3/week-2/lesson.md:424]: I like this!

**Review by kfklein15 — DISMISSED** [review]
Ok with Daniel's changes.

File: docs/js-core-3/week-2/lesson.md (hunk 1 above)
Comment (esma-g) [reply docs/js-core-3/week-2/lesson.md]: Yes, sorry.

**Pull request closed without merging**